third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -1453,11 +1453,8 @@ struct TritonAMDGPUAccelerateMatmulPass
           /*benefit=*/2);
       break;
     case ISAFamily::RDNA3:
-      // Only gfx12 is supported for now
-      if (getWmmaVersion(archGenerationName) == 2) {
-        ttg::populateDecomposeScaledBlockedPatterns(mfmaPatterns,
-                                                    /*benefit=*/3);
-      }
+      ttg::populateDecomposeScaledBlockedPatterns(mfmaPatterns,
```

Comment:
I see we are still putting both gfx11 and gfx12 under RDNA3 due to historical reasons. Not for this pull request but can you follow up later to define `ISAFamily::RDNA4` properly and update the codebase wherever needed?

Reply:
Yes. Will do!